### PR TITLE
Modified/added to existing citations

### PR DESCRIPTION
**Pull request by lynnpepin**
I modified a few citations to use more up-to-date / authoritative sources.

### CODE REVIEW

**Comment by leonoverweel** [comment]
Thanks, does this overlap a bit with your previous PR? Should we just work/merge on this one?

**Comment by lynnpepin** [comment]
Yep - it branches off that one. I didn't remove the errant spaces in this yet -- I can remove the spaces from this one and then submit a new PR, would that be good?

**Comment by leonoverweel** [comment]
Yes, just push to this `fix-citations` branch and it should show up on this PR, no need for a new one. I'll close #2 then.

**Comment by lynnpepin** [comment]
Done! (Thanks for letting me know -- I've not made edits after a PR before.)

File: README.md [hunk 1]
```diff
@@ -41,6 +41,47 @@ It may also be a good idea to add the specific version of Python, for example fo
 
 Packages are listed alphabetically. If this list grows too long, maybe we can add categories.
 
+### Caffe BibTeX citation
+
+```bibtex
+@article{jia2014caffe,
```
Comment (leonoverweel) [inline README.md]: Change this reference name to just `caffe`

File: README.md [hunk 2]
```diff
@@ -41,6 +41,47 @@ It may also be a good idea to add the specific version of Python, for example fo
...
+  Year={2014}
+}
+```
+
+([source](https://caffe.berkeleyvision.org/))
+
+### CoreNLP BibTeX citation
+
+```bibtex
+@InProceedings{manning-EtAl:2014:P14-5,
```
Comment (leonoverweel) [inline README.md]: Change this to just `corenlp`

File: README.md [hunk 3]
```diff
@@ -41,6 +41,47 @@ It may also be a good idea to add the specific version of Python, for example fo
...
+  url={http://www.aclweb.org/anthology/P/P14/P14-5010}
+}
+```
+
+([source](https://stanfordnlp.github.io/CoreNLP/citing.html))
+
+### Keras BibTeX citation
+
+```bibtex
+@misc{chollet2015keras,
```
Comment (leonoverweel) [inline README.md]: Change this to just `keras`

File: README.md [hunk 4]
```diff
@@ -90,14 +133,14 @@ Packages are listed alphabetically. If this list grows too long, maybe we can ad
 
 ```bibtex
 @article{opencv_library,
```
Comment (leonoverweel) [inline README.md]: Change this to just `opencv`

File: README.md [hunk 5]
```diff
@@ -116,20 +159,20 @@ Packages are listed alphabetically. If this list grows too long, maybe we can ad
...
 
-([source](https://github.com/pandas-dev/pandas/issues/24036))
+([source](https://pandas.pydata.org/talks.html))
 
 ### PyTorch BibTeX citation
 
 ```bibtex
-@inproceedings{pytorch,
-  title={Automatic differentiation in PyTorch},
+@inproceedings{paszke2017automatic,
```
Comment (leonoverweel) [inline README.md]: Change this to just `pytorch`

File: README.md [hunk 6]
```diff
@@ -232,6 +275,24 @@ Packages are listed alphabetically. If this list grows too long, maybe we can ad
 
 ([source](https://www.tensorflow.org/about/bib))
 
+### Theano BibTeX citation
+
+```bibtex
+@ARTICLE{2016arXiv160502688short,
```
Comment (leonoverweel) [inline README.md]: Change this to `@article{theano,` (lower-case "article")

**Review by leonoverweel — CHANGES_REQUESTED** [review]
No problem and thanks! I requested just a few more small changes to make the reference naming consistent with the rest of the repo. I can also merge and make them myself if that's easier, whatever works for you.

**Comment by lynnpepin** [comment]
Yea, you can go ahead and make them! I don't know if it's a point of GitHub etiquette or not, but I have no stake in the matter. I also modified them on my PR if that's easier. (I changed theano as well.)

**Review by leonoverweel — APPROVED** [review]
Perfect, merging.

**Pull request merged**